packages/app_ui/lib/src/dialogs/bottom_animated_dialog.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class BottomAnimatedDialog {
+  static void show(BuildContext context, {required Widget child}) {
+    showGeneralDialog(
+      barrierLabel: 'BottomAnimatedDialog',
+      barrierDismissible: true,
+      barrierColor: Colors.black.withOpacity(0.5),
+      transitionDuration: const Duration(milliseconds: 500),
+      context: context,
+      pageBuilder: (context, anim1, anim2) {
+        return Align(
+          alignment: Alignment.bottomCenter,
+          child: Container(
+            height: 300,
+            width: double.infinity,
+            margin: EdgeInsets.only(bottom: 20, left: 12, right: 12),
```

Comment:
ussualy we don't want to use hardcoded values anywhere in the code.
In case of margins, paddings, spacings and simmilar we can use something called AppSizes.
Let's go to your app_ui package, create new folder named sizes, and create new file with the class called AppSizes, that takes different static const, e.g: static const s12 = 12;
Then you can use such a const in every place where you need specific pixels value. Sorry that I didn't mentioned that before. It's just a good standard.